rubin_scheduler/scheduler/features/conditions.py

```diff
@@ -572,7 +520,7 @@ def az_to_antisun(self):
         return self._az_to_antisun
 
     def __repr__(self):
-        return f"<{self.__class__.__name__} mjd_start='{self.mjd_start}' at {hex(id(self))}>"
+        return f"<{self.__class__.__name__} at {hex(id(self))}>"
```

Comment:
It might be useful to have this print the current time (not sure why it was mjd_start before, but conditions.mjd might be a good shortcut).